logstash-core/src/main/java/org/logstash/instrument/witness/pipeline/PluginsWitness.java

```diff
@@ -89,8 +89,8 @@ public void forgetAll() {
      * @param id     the id of the plugin
      * @return existing or new {@link PluginWitness}
      */
-    private PluginWitness getPlugin(Map<String, PluginWitness> plugin, String id) {
-        return plugin.computeIfAbsent(id, k -> new PluginWitness(k));
+    private static PluginWitness getPlugin(Map<String, PluginWitness> plugin, String id) {
+        return plugin.computeIfAbsent(id, PluginWitness::new);
```

Comment:
Was this actually a compiler warning, or just an IntelliJ default inspection warning ?

Reply:
@jakelandis hah seems like this was an idea bug, it didn't understand the type of `k` correctly in the EAP. Still much nicer this way imo :P